ClosedXML/Excel/ConditionalFormats/XLConditionalFormat.cs

```diff
@@ -39,10 +41,11 @@ public XLConditionalFormat(XLConditionalFormat other)
             ShowBarOnly = other.ShowBarOnly;
         }
 
+        public String Name { get; set; }
```

Comment:
Will the user ever set the `Name` manually? If not, make the `set` internal.

Reply:
Normaly not, but how make set for test examples?

Reply:
The `ClosedXML_Examples` project should contain only code samples that a normal user can execute.